lib/ecs_deploy/configuration.rb

```diff
@@ -6,7 +6,9 @@ class Configuration
       :secret_access_key,
       :default_region,
       :deploy_wait_timeout,
-      :ecs_service_role
+      :ecs_service_role,
+      :ecs_service_wait_until_max_attempts,
```

Comment:
[imo] I prefer `ecs_wait_until_services_stable_max_attempts` and `ecs_wait_until_services_stable_delay`

Reply:
It looks good to me. I'll change it.